easybuild/easyconfigs/l/libbeef/libbeef-0.1.1-intel-2015a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'libbeef'
+version = '0.1.1'
+
+homepage = 'https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software'
+description = """libbeef is a package providing the class of BEEF-vdW functionals and
+ Bayesian error estimates to Fortran- or C-based DFT codes."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://confluence.slac.stanford.edu/download/attachments/146704476']
+sources = [SOURCE_TAR_GZ]
+
+configopts = 'CC="$CC"'
+
+sanity_check_paths = {
+    'files': ['bin/bee', 'lib/libbeef.a'],
```

Comment:
The binary is named `bee`, the library is named `beef`. I don't question, I just observe. ;-)